### ISSUE TITLE
Pin no mapa não aparece no local certo

### ISSUE THREAD

**Issue by thaisrigolon** [lafabbricabr/mapasculturaisThemeHortas#17]
O pin no mapa não está aparecendo no endereço correto:

![1](https://cloud.githubusercontent.com/assets/18294515/20983649/63016d30-bca4-11e6-8a08-6079280986f3.png)

**Comment by GustavoVS** [comment]
Bug referente ao "core" do mapas

**Comment by thaisrigolon** [comment]
Certo, mas vamos manter a issue aberta por aqui, ok? Assim não esquecemos disso.
Sabe se tem alguém vendo esse problema? Será que tem issue sobre isso?
